package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopware-ag/acceptance-test-suite",
-  "version": "3.11.0",
+  "version": "3.12.1",
```

Comment:
Version should be either 3.11.4 or 4.0.0. Question is, if you need this in both versions, how would you do it?